eventuate-local-java-cdc-connector-test-util/src/main/java/io/eventuate/local/test/util/TestHelper.java

```diff
@@ -212,6 +221,24 @@ public void waitForEventInKafka(KafkaConsumer<String, byte[]> consumer, String e
     throw new RuntimeException("entity not found: " + entityId);
   }
 
+  //TODO: Use in all tests where is possible.
+  public MessageAssertion prepareBinlogEntryHandlerMessageQueue(BinlogEntryReader binlogEntryReader) {
+    ConcurrentLinkedQueue<MessageWithDestination> messages = new ConcurrentLinkedQueue<>();
```

Comment:
`ConcurrentLinkedQueue<MessageWithDestination> messages` should be an implementation detail of MessageAssertion.

The callback should simply call messageAssertion.add()